app/src/main/java/org/liberty/android/fantastischmemo/ui/StatisticsScreen.java

```diff
@@ -246,6 +257,8 @@ public Chart generateChart(BarData data) {
             chart.getXAxis().setTextColor(Color.WHITE);
             chart.getAxisLeft().setTextColor(Color.WHITE);
             chart.getAxisRight().setTextColor(Color.WHITE);
+            chart.getAxisLeft().setValueFormatter(new ChartYAxisValueFormatter());
```

Comment:
I'm not sure if we want to have the formatters as static variable in the StatisticsScreen class (as it is not possible to put them in the inner class ChartTask). I changed the Formatter inside the `ChartYAxisValueFormatter` and `ChartValueFormatter` to a static member, and now the instantiation cost for the both formatters should be minimal. 

However, it is a trade-off in coupling and memory efficiency. If you think it is better to make them a static member for `StatisticsScreen`, I'll change it this way.